### DISCUSSION TITLE
feature request: unlimited rides and map size

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23060] [Feature requests]
Hi

My feature request is easy but I don't know if it's easy to implement.
Considering today's crazy fast cpus and amounts of ram, would it be possible to implement unlimited rides and map size?

**Comment by telk5093** [comment]
Unlimited rides and map size requires unlimited cpu and memory specs, hence that's impossible.

**Comment by ZehMatt** [comment]
No, even if you had an infinite amount of memory it would also infinitely raise the complexity in the simulation.

**Comment by ghost** [comment]
What are the current limits then for rides and map size and why is this limit chosen?
It must be at least 1000x the limit from the original game? We now have GBs of RAM while back then not even 1 GB.

**Reply by sjoerddebruin** [reply]
Have you tried filling a park with the current limits? Surely you will encounter some slowdowns, run out of creativity or see weird guest behaviour.

**Reply by ghost** [reply]
But that has nothing to do with hardware limits?

**Comment by Realsteel89** [comment]
> Hi
> 
> My feature request is easy but I don't know if it's easy to implement.
> Considering today's crazy fast cpus and amounts of ram, would it be possible to implement unlimited rides and map size?
> 

I think now the list of unlimed rides is more than all active rides you can build with ctr content into the game at this moment.
